frontend/src/properties/propertysections/WidgetContainerPropertySection.types.ts

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { WidgetContainer, WidgetSubscription } from 'form/Form.types';
+import { Selection } from 'workbench/Workbench.types';
+
+export interface WidgetContainerPropertySectionProps {
+  editingContextId: string;
+  formId: string;
+  widget: WidgetContainer;
+  widgetSubscriptions: WidgetSubscription[];
+  setSelection: (selection: Selection) => void;
+  readOnly: boolean;
+}
+
+export interface GQLErrorPayload extends GQLUpdateWidgetFocusPayload {
+  message: string;
+}
+
+export interface GQLUpdateWidgetFocusMutationData {
+  updateWidgetFocus: GQLUpdateWidgetFocusPayload;
+}
+
+export interface GQLUpdateWidgetFocusPayload {
+  __typename: string;
+}
+
+export interface GQLUpdateWidgetFocusMutationVariables {
+  input: GQLUpdateWidgetFocusInput;
+}
+
+export interface GQLUpdateWidgetFocusInput {
+  id: string;
+  editingContextId: string;
+  representationId: string;
+  widgetId: string;
+  selected: boolean;
+}
+
+export interface GQLUpdateWidgetFocusSuccessPayload extends GQLUpdateWidgetFocusPayload {}
```

Comment:
remove the success payload since it is not used

Reply:
Done